examples/trigonometry.R

```diff
@@ -0,0 +1,71 @@
+# https://processing.org/tutorials/trig/
+
+px <- 0
+py <- 0
+px2 <- 0
+py2 <- 0
+angle <- 0
+angle2 <- 0
+x <- 0
+x2 <- 0
+radius <- 50
+frequency <- 2
+frequency2 <- 2
```

Comment:
Long style is fine for now. Having tried it out, I'm not sure that the concise R style is more readable, honestly.